docs/pr-quality/pr-format.md

```diff
@@ -2,28 +2,24 @@
 - All PR's at Colonist follow a general structure as mentioned below; edit accordingly. A reviewer should be able to have all the information they require directly avaiable in the PR without having to go out and look for it themselves
 
 ```md
-<!-- DELETE THE PARTS YOU DON'T USE -->
+<!-- EDIT THE FOLLOWING -->
 ## Summary
 * my_summary_pls
 
 ## Details
 * edit_me_pls
 
 ## Testing
-* explain_how_qa_should_test_this_PR
-* list_edge_cases_qa_should_extra_pay_attention_to:
+* explain_how_reviewer_should_test_this_PR
+* list_edge_cases_to_pay_extra_attention_to:
 - [ ] edge_case_1
 - [ ] edge_case_2
+
 #### Platforms affected by this PR:  
 <!-- DO NOT DELETE OR EDIT THIS LIST -->
 - [ ] Web
-- [ ] Mobile
-  - [ ] Portrait
-  - [ ] Landscape
-- [ ] Discord
-  - [ ] Browser
-  - [ ] Desktop
-  - [ ] Mobile
+  - [ ] Full screen
+  - [ ] Small screen
```

Comment:
Why these?

Reply:
Demi mentioned these 2 in his review https://github.com/colonistio/developer-checklist/pull/4#issuecomment-2275372292 but I may have taken them too literally. I'm thinking of replacing both with 'Responsive'

Reply:
Makes sense, we can keep it then